test/test_load.rb

```diff
@@ -41,4 +44,32 @@ def test_debug_load
   ensure
     Debugger.stop if Debugger.started?
   end
+
+  module MyBootsnap
+    def load_iseq(path)
+      iseq = RubyVM::InstructionSequence.compile_file(path)
+
+      Debugger.unset_iseq_flags(iseq)
+      iseq
+    end
+  end
+
+  def test_bootsnap
+    @@at_line = nil
+    src_dir = File.dirname(__FILE__)
+    prog_script = File.join(src_dir, 'example', 'bootsnap', 'bootsnap.rb')
+
+    class << RubyVM::InstructionSequence
+      prepend MyBootsnap
+    end
+    bt = Debugger.debug_load(prog_script, true)
+    assert_equal(nil, bt)
+    assert_not_nil(@@at_line)
+    if RUBY_VERSION >= '2.5'
+      assert_equal(['debase.rb', 101], @@at_line)
+    end
+
+    assert(Debugger.started?)
+    Debugger.stop
```

Comment:
You should remove `load_iseq` monkey patch after test ends

https://stackoverflow.com/questions/11894308/when-to-use-undef-method-and-when-to-use-remove-method